torch/CMakeLists.txt

```diff
@@ -337,11 +337,17 @@ endif()
 
 # in case of the split build we need to add compile definitions
 if(BUILD_LIBTORCHLESS)
+
+  if(USE_UCC)
+    target_link_libraries(torch_python PRIVATE __caffe2_ucc)
```

Comment:
Is linking with `__caffe2_ucc` here related/intentional?

Reply:
this is just so that __caffe2_ucc is linked to torch_python in the split build (by default its linked to torch_cuda which then gets propogated to torch_python)